src/webgpu/api/validation/createBindGroupLayout.spec.ts

```diff
@@ -298,19 +298,19 @@ const kMaxResourcesCases = kUnitCaseParamsBuilder
   .combine('extraVisibility', kShaderStages)
   .filter(p => (bindingTypeInfo(p.extraEntry).validStages & p.extraVisibility) !== 0);
 
-// Should never fail unless kLimitInfo.maxBindingsPerBindGroup.default is exceeded, because the validation for
-// resources-of-type-per-stage is in pipeline layout creation.
+// One bind group layout can have a maximum number of each type of binding *per stage* (which is
+// different for each type). Test that the max works, then add one more entry of same-or-different
+// type and same-or-different visibility.
 g.test('max_resources_per_stage,in_bind_group_layout')
   .desc(
     `
     Test that the maximum number of bindings of a given type per-stage cannot be exceeded in a
     single bind group layout.
     - Test each binding type.
     - Test that creation of a bind group layout using the maximum number of bindings works.
-    - Test that creation of a bind group layout using the maximum number of bindings + 1 fails.
-    - TODO(#230): Update to enforce per-stage and per-pipeline-layout limits on BGLs as well.`
```

Comment:
This test hasn't actually changed. Am I reading right that the test was already up-to-date despite, just outdated comment and TODO?